### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nafizh**
In the latest pytorch, grad data is not created until loss.backward() is called once. So, it throws an error without the condition.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,8 +208,9 @@ for t in range(500):
   print(t, loss.data[0])
   
   # Manually zero the gradients before running the backward pass
-  w1.grad.data.zero_()
-  w2.grad.data.zero_()
+  if t:
```
Comment (hunkim) [inline README.md]: In the pytorch.org tutorial, `*zero_()` is called after `backward()`. But in this example, w.grade.Data can be None.

**Comment by jcjohnson** [comment]
Fixed in https://github.com/jcjohnson/pytorch-examples/commit/0d8adca7c5ef427c738fe3a484edc76ab6ddc105.
